From 665ef7f0211de7b83a38f86d746da51793496e77 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?J=C3=B6rg=20Sommer?= <joerg@alea.gnuu.de>
Date: Thu, 21 Aug 2008 11:22:26 +0200
Subject: [PATCH 1/5] Improve variable assignment

The variable expansion ${X:-Y} means expand $X if $X is set and not
empty, else expand Y. This is exactly what the former expression did.
---
 getbuildlog |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/getbuildlog b/getbuildlog
index 59768e9..daa0910 100755
--- a/getbuildlog
+++ b/getbuildlog
@@ -65,8 +65,8 @@ if [ ! -x "`which wget 2>/dev/null`" ]; then
 fi
 
 PACKAGE=$1
-VERSION=`(test -z "$2" && echo "[:~+.[:alnum:]-]+") || echo "$2"`
-ARCH=`(test -z "$3" && echo "[[:alnum:]-]+") || echo "$3"`
+VERSION=${2:-[:~+.[:alnum:]-]+}
+ARCH=${3:-[[:alnum:]-]+}
 ESCAPED_PACKAGE=`echo "$PACKAGE" | sed -e 's/\+/\\\+/g'`
 
 PATTERN="fetch\.(cgi|php)\?&pkg=$ESCAPED_PACKAGE&ver=$VERSION&arch=$ARCH&\
-- 
1.5.6.3

